examples/small-scale-udn-l2.yaml

```diff
@@ -30,8 +30,8 @@ tests :
       direction: 1
       threshold: 10
 
-    - name:  ovsCPU
-      metricName : cgroupCPUSeconds-Workers
+    - name:  ovsCPU-Workers
+      metricName.keyword: cgroupCPUSeconds-Workers
```

Comment:
The agg_type here is avg. We should only have a single datapoint returned, but it should be 'max' to be consistent.